source/content/guides/object-cache/02-enable-object-cache.md

```diff
@@ -135,7 +135,7 @@ This configuration uses the `Redis_CacheCompressed` class for better performance
 
   ```php:title=settings.php
   // All Pantheon Environments.
-  if (defined('PANTHEON_ENVIRONMENT')) {
+  if (!empty($_ENV['CACHE_HOST'])) {
```

Comment:
Should we maybe check both `!empty($_ENV['PANTHEON_ENVIRONMENT'])` and `!empty($_ENV['CACHE_HOST'])`?

Reply:
Looks like this is the logic that's used for Wordpress (line 38 of this fine), so if we want to keep the `PANTHEON_ENVIRONMENT` check, might as well have it in both CMSs..